src/state/opamFormatUpgrade.ml

```diff
@@ -1008,7 +1008,11 @@ let from_2_0_beta_to_2_0_beta5 root conf =
        (OpamFile.Config.eval_variables conf))
     conf
 
-let latest_version = v2_0_beta5
+let v2_0 = OpamVersion.of_string "2.0"
+
+let from_2_0_beta5_to_2_0 _ conf = conf
+
+let latest_version = v2_0
```

Comment:
I know we're *really* close to release, but is that wholly safe? It's OK having `opam-version: "2.0"` not have a final meaning until actually released, but this could result in having the opamroot version having to be greater than the release version for 2.0?!

Reply:
(I'm thinking of OCaml, where bumping versions and so forth is always the last commit before release it tagged)